README.md

```diff
@@ -74,34 +79,36 @@ client.on('message', function (topic, payload) {
 });
 ```
 
-### SSE
+### SSE with String Payload
 
 ```
-var eventSource = new EventSource(<url>);
-var channelDecoder = new DeltaCodec.VcdiffDecoder();
-eventSource.onmessage = function(event) {
-	var message = JSON.parse(event.data);
-	var data = message.data;
-	try {
-		if (DeltaCodec.VcdiffDecoder.isDelta(data, true)) {
-			data = channelDecoder.applyDelta(data, undefined, undefined, true).asUtf8String();
-		} else {
-			channelDecoder.setBase(data);
-		}
-	} catch(e) {
-		/* Delta decoder error */
-	}
-
-	/* Process decoded data */
-	console.log(data);
-};
+    const eventSource = new EventSource(<url>);
```

Comment:
The indentation of this code doesn't match the MQTT example (this one is further over to the right).